addons/share-through-mystuff/userscript.js

```diff
@@ -1,4 +1,11 @@
 export default async function ({ addon, console, msg }) {
+  Scratch.MyStuff.ProjectThumbnailCollectionView.prototype.shared = function (project) {
+    // Scratch's implementation incorrectly calls fadeOut on the entire list
+    if (this.model.options.collectionType === "notshared") {
+      project.$el.fadeOut();
+    }
+  };
+
```

Comment:
Good catch here. I don't think we tested sharing projects with the "unsharing projects" tab selected.

Reply:
The addon already includes a fix for this, but it's more complex than the one I added in this PR.